lib/Demangling/NodePrinter.cpp

```diff
@@ -1414,6 +1416,7 @@ NodePointer NodePrinter::print(NodePointer Node, unsigned depth,
   case Node::Kind::CFunctionPointer:
   case Node::Kind::ObjCBlock:
   case Node::Kind::EscapingObjCBlock:
+  case Node::Kind::BackDeploymentFallback:
```

Comment:
Mangling output should differentiate the fallback from the original function I think

Reply:
Makes sense, I'll figure out how to accomplish that.

Reply:
Fixed and added tests for the mangling classification. This revealed a number of issues with my existing mangling implementation which are now fixed.